sql/transaction.sql

```diff
@@ -15,4 +15,10 @@ CREATE TABLE transaction (
     debit_ref VARCHAR(50) NOT NULL,
     transaction_status transaction_status_enum NOT NULL
 );
--- rollback DROP TABLE transaction
\ No newline at end of file
+-- rollback DROP TABLE transaction
+
+-- changeset devwasabi2:alter-transaction_amount
+ALTER TABLE transaction
+ALTER COLUMN transaction_amount TYPE BIGINT USING transaction_amount::BIGINT,
+ALTER COLUMN transaction_amount SET NOT NULL;
+-- rollback ALTER TABLE transaction DROP COLUMN transaction_amount
```

Comment:
Maybe make it easier and just change the main table and we will just recreate the db from scratch